src/main/java/uk/gov/hmcts/reform/idam/health/probe/ScheduledHealthProbeIndicator.java

```diff
@@ -51,20 +57,25 @@ protected void refresh() {
         boolean probeResult = this.healthProbe.probe();
 
         if (probeResult || failureHandling == HealthProbeFailureHandling.MARK_AS_DOWN) {
-            if (log.isInfoEnabled()) {
-                if ((probeResult) && (this.status != Status.UP)) {
-                    log.info("{}: Status changing from {} to UP", this.healthProbe.getName(), this.status);
-                } else if ((!probeResult) && (this.status == Status.UP)) {
-                    log.info("{}: Status changing from UP to DOWN", this.healthProbe.getName());
-                }
-            }
+            printLogMessage(probeResult);
+
             this.status = probeResult ? Status.UP : Status.DOWN;
             this.statusDateTime = LocalDateTime.now(clock);
         } else if (failureHandling == HealthProbeFailureHandling.IGNORE) {
             log.warn("{}: DOWN state ignored", this.healthProbe.getName());
         }
     }
 
```

Comment:
minor: if you make this method return the new status, you could greatly simplify the code here
i.e Status newStatus = probeResult ? Status.UP : Status.DOWN;
(if logInfo and status != this.status) print message;
return newStatus;

Reply:
Returning a status in a method that is called "printLogMessage" it sounds misleading. The logging depends on the status change, so comparing newStatus and oldStatus it makes sense. I'll update it.